modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/sort/DefaultSortTranslator.java

```diff
@@ -69,7 +69,12 @@ public void translate(
 
 			SortOrder sortOrder = SortOrder.ASC;
 
-			if (sort.isReverse() || sortFieldName.equals("_score")) {
+			String originalSortName = sort.getFieldName();
+
+			if (sort.isReverse() ||
+				(sortFieldName.equals("_score") &&
+				 !sortFieldName.equals(originalSortName))) {
```

Comment:
Remove this check - instead, add "_score" here as a similar no-op return:

https://github.com/liferay/liferay-portal/blob/2dde8068d881bf64213e055d12470148c9b5bf4b/modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/sort/DefaultSortTranslator.java#L128-L130